src/actions/plugins.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {PluginTypes} from '../action_types';
+import {Client4} from '../client';
+
+import {ActionFunc} from '../types/actions';
```

Comment:
Are we able to do these imports as absolute imports, rather than relative paths? Same with other files